packages/client/tests/functional/issues/18846-empty-array/_matrix.ts

```diff
@@ -0,0 +1,24 @@
+import { defineMatrix } from '../../_utils/defineMatrix'
+
+export default defineMatrix(() => [
+  [
+    {
+      provider: 'sqlite',
+    },
+    {
+      provider: 'postgresql',
+    },
+    {
+      provider: 'mysql',
+    },
+    {
+      provider: 'mongodb',
+    },
+    {
+      provider: 'cockroachdb',
+    },
+    {
+      provider: 'sqlserver',
+    },
+  ],
+])
```

Comment:
Technically, we could only test on 1 database? (Is there something interesting to test on different databases for this?)

Reply:
I think generally our approach for functional tests have always been "test on all providers unless you have really good reasons not too", because we had suprirsing failures on selected providers where we were absolutely sure provider makes no difference